### ISSUE TITLE
Mobile Device Support?

### ISSUE THREAD

**Issue by ofusion** [stillwwater/command_terminal#10]

**Comment by Wokarol** [comment]
I've just tested few things and it looks like it supports mobile very well, with just one exception... buttons for showing console are almost unclickable on FHD 5.1'' phone.

**Comment by Wokarol** [comment]
And some kind of auto-completion button for mobiles would be cool addition too

**Comment by ofusion** [comment]
I see all controls use KeyboradEvents, how it is usable on mobile device?

**Comment by Wokarol** [comment]
Simply tapping on input field brings up keyboard.

**Comment by ofusion** [comment]
Yes, I know that, but you should open terminal first, Normally I would like a special tap action on some specified screen position to open terminal rather then keep terminal showing all the time using "ShowGUIButtons" option.

**Comment by stillwwater** [comment]
I can't add a special tap action as it might interfere with controls on some games. You can implement custom functionality for enabling the console using `Terminal.SetState`:

```csharp
void Start() {
    var terminal = GetComponent<Terminal>(); // Reference to terminal component

    terminal.SetState(TerminalState.OpenSmall); // Open small terminal window
    terminal.SetState(TerminalState.OpenFull);  // Open full terminal window
    terminal.SetState(TerminalState.Close);     // Close terminal window

    terminal.ToggleState(); // Shortcut for switching between opened and closed windows
}
```

> And some kind of auto-completion button for mobiles would be cool addition too

I agree I'll look into this.

**Comment by Wokarol** [comment]
I think that adding GUI scaling for terminal would be good because on FHD Phone everything is small, and on QHD Phone (1440p) everything is barely readable
